source/examples/generated/web/authentication-log-in-and-out.test.snippet.builtin-apple-oauth.js

```diff
@@ -4,11 +4,11 @@ const redirectUri = "https://app.example.com/handleOAuthLogin";
 const credentials = Realm.Credentials.apple(redirectUri);
 
 // Calling logIn() opens an Apple authentication screen in a new window.
-app.logIn(credentials).then((user) => {
-  // The logIn() promise will not resolve until you call `handleAuthRedirect()`
-  // from the new window after the user has successfully authenticated.
-  console.log(`Logged in with id: ${user.id}`);
-});
+const user = app.logIn(credentials);
```

Comment:
```suggestion
const user = await app.logIn(credentials);
```

Do we need to await the Promise?